src/public/modules/users/controllers/authentication.client.controller.js

```diff
@@ -239,25 +246,32 @@ function AuthenticationController(
         }, 500)
 
         GTag.loginSuccess('otp')
-      },
-      function (error) {
+      })
+      .catch((error) => {
+        let errorMsg = get(
+          error,
+          'response.data',
+          'Failed to send login OTP. Please try again later and if the problem persists, contact us.',
+        )
+        const errorStatus = get(error, 'response.status')
+        if (errorStatus >= StatusCodes.INTERNAL_SERVER_ERROR) {
+          errorMsg = 'An unknown error occurred'
+        }
```

Comment:
idk why we override the message returned by the server, but we can just preserve existing behaviour

Reply:
Yea, didn't want to touch existing behaviour (for now), can be fixed in rEaCt